src/components/Dropdown/Dropdown.tsx

```diff
@@ -136,7 +140,7 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
 
       <ScreenReader
         announcementKey={screenReaderKey}
-        announcementText={isActive ? screenReaderText : ''}
+        announcementText={isActive && (hasTyped || items.length || value) ? screenReaderText : ''}
```

Comment:
is it possible for `hasTyped` to be false but `value` to be true? I'm thinkin we can remove value here but you just added it so I might have missed something

Reply:
Yeah Nidhi pointed out that in some cases there might already be something put in the search bar input by default before the user interacts with it, and in that case we still want the autocomplete to be read